src/main/java/sysc4806/graduateAdmissions/GraduateAdmissions.java

```diff
@@ -30,11 +30,11 @@ public static void main(String[] args) {
     public CommandLineRunner demo(BuddyInfoRepository repository) {
         return (args) -> {
             // save a few customers
-            repository.save(new BuddyInfo("Jack", "address", "number", 1));
-            repository.save(new BuddyInfo("Chloe", "O'Brian", "number", 1));
-            repository.save(new BuddyInfo("Kim", "Bauer", "number", 1));
-            repository.save(new BuddyInfo("David", "Palmer", "number", 1));
-            repository.save(new BuddyInfo("Michelle", "Dessler", "number", 1));
+            repository.save(new BuddyInfo("Jack", "address1", "number", 1));
+            repository.save(new BuddyInfo("Chloe", "123 eagfege", "number", 1));
+            repository.save(new BuddyInfo("Kim", "245 aefeg", "number", 1));
+            repository.save(new BuddyInfo("David", "23 grhgrhr", "number", 1));
+            repository.save(new BuddyInfo("Michelle", "980 sdfgeg", "number", 1));
```

Comment:
Doesn't need to be changed in this PR but an issue should be created to remove all the BuddyInfo and AddressBook stuff